test/escort.test.js

```diff
@@ -15,1586 +15,1702 @@ var makeBadString = (function (Ctor) {
     };
 }(String));
 
-module.exports = {
-    "methods static": function () {
-        var url;
-        var app = connect(
-            escort(function (routes) {
-                url = routes.url;
-                
-                methods.forEach(function (method) {
-                    routes[method]("home_" + method, "/" + method, function (req, res) {
-                        res.end(method.toUpperCase() + " /" + method);
-                    });
-                });
-            })
-        );
-        
+function makeConnect(...apps) {
```

Comment:
`connect` changed its method signature, instead of `connect(app)` you need to do `a = connect(); a.use(app)`.